### PR TITLE
corrfunc

### PR DESCRIPTION
**Pull request by beckermr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/corrfunc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+test:
+  imports:
+    - Corrfunc
+
+about:
+  home: https://github.com/manodeep/Corrfunc
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (scopatz) [inline recipes/corrfunc/meta.yaml]: This doesn't seem right :wink:

Comment (beckermr) [reply recipes/corrfunc/meta.yaml]: LOL - still working here but yes!

Comment (beckermr) [reply recipes/corrfunc/meta.yaml]: fixed!

**Comment by beckermr** [comment]
@conda-forge/staged-recipes @scopatz This one is ready to go!

**Pull request merged**